DarkCodex/DarkCodex.csproj

```diff
@@ -102,7 +102,8 @@
 		</ItemGroup>
 
 		<Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(ModFolder)\$(TargetName)\%(RecursiveDir)" />
-		<Exec Condition="'$(Configuration)'=='Release'" WorkingDirectory="$(ModFolder)\" Command="&quot;$(Zip7)&quot; u &quot;-x!*.zip&quot; &quot;$(TargetName)\$(TargetName).zip&quot; @&quot;$(TargetName)\filelist.txt&quot; &gt;NUL" />
+		<!--<Exec Condition="'$(Configuration)'=='Release'" WorkingDirectory="$(ModFolder)\" Command="&quot;$(Zip7)&quot; u &quot;-x!*.zip&quot; &quot;$(TargetName)\$(TargetName).zip&quot; @&quot;$(TargetName)\filelist.txt&quot; &gt;NUL"/>-->
+		<!--You don't actually include the needed filelist.txt for the above to execute on anyone else's machine.-->
```

Comment:
I see no reason for anyone (other than me) to compile in release. This mod is not abandoned. If it does come up, just look what files are in the previous release. Or zip it manually.